app/services/member-consequences.js

```diff
@@ -3,6 +3,54 @@ import Ember from 'ember';
 export default Ember.Service.extend({
   store: Ember.inject.service(),
   sessionStorage: Ember.inject.service('session'),
+  calculatePreg35(member, chapter, tags) {
+
+    // Get preg35 option, if it exists
+    var preg35_option = this.get('store').peekAll('option').filterBy('text', '__input-preg35').objectAt(0);
```

Comment:
Refactor to use the `slug` property so that the question can be directly referenced instead of going through the `option`.